{{cookiecutter.project_name}}/Makefile

```diff
@@ -210,29 +210,35 @@ test-all: tests
 tests: depends-ci .clean-test
 	$(NOSE) $(PACKAGE) tests $(NOSE_OPTS) -xv
 ifndef TRAVIS
-	$(COVERAGE) html --directory htmlcov --fail-under=$(COMBINED_TEST_COVERAGE)
+	$(COVERAGE) report --show-missing --fail-under=$(COMBINED_TEST_COVERAGE)
 endif
 {% elif cookiecutter.test_runner == "pytest" %}
-PYTEST_OPTS := --doctest-modules --cov=$(PACKAGE) --cov-report=term-missing --cov-report=html
+PYTEST_OPTS := --doctest-modules --cov=$(PACKAGE) --cov-report=term-missing --no-cov-on-fail
 
 .PHONY: test-unit
 test-unit: test
 .PHONY: test
 test: depends-ci .clean-test
 	$(PYTEST) $(PYTEST_OPTS) $(PACKAGE)
-	$(COVERAGE) report --fail-under=$(UNIT_TEST_COVERAGE) > /dev/null
+ifndef TRAVIS
```

Comment:
I'm not a fan of `ifdef`-ing out travis stuff everywhere. The commands in this block have no usefulness when running on travis, but they don't take long to run either and it's more readable without the `ifdefs`